Fix application template link

mark-dce opened the pull request:
Link needs to point to 'projecthydra-labs' instead of 'projecthydra'
https://raw.githubusercontent.com/projecthydra-labs/hyrax/master/template.rb

mark-dce commented:
Wow - looks like I triggered 900+ RuboCop errors by fixing a typo in the Readme ;)

jcoyne commented:
That'll teach you.

jcoyne commented:
rebase?

mark-dce commented:
Literally the only file I changed is README.md.  CI is failing on RuboCop on files I never touched.  Is it possible RuboCop defaults got changed in some other way?

> Running RuboCop...
Inspecting 902 files
......................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................C....CCC....C..CCCCC.C...C..C.CCCC..CCCC.C.C.......C.C.CC...........................................................C........CC.C.........C......CC.C.C....C....C..............C..CCC................................C.....C......C.......C..........................C..C.C.......................C......C.....C.C.CC......C................................................C..................................C...........C....
Offenses:
spec/actors/hyrax/actors/add_as_member_of_collections_actor_spec.rb:25:29: C: RSpec/MessageSpies: Prefer have_received for setting message expectations. Setup root_actor as a spy using allow or instance_spy.
      expect(root_actor).to receive(:create).with(title: ['test'])
                            ^^^^^^^

...

> 902 files inspected, 161 offenses detected
RuboCop failed!

jcoyne commented:
@mark-dce did you rebase it? The master branch should ensure you do not have that problem.

mjgiarlo commented:
Thanks for this, @mark-dce! Approved.

mark-dce commented:
@jcoyne @mjgiarlo Ok, I pulled a clean copy of the repo 30 minutes ago, checked out the typo branch, rebased master, force pushed the branch and am still getting errors.  I'm 99% sure the README change did not cause the "Hyrax::Statistics::Depositors::Summary#depositors when dates are present gathers user deposits during a date range"  Failure/Error in the CI run.

This is a pretty frustrating process to try to help improve an obvious and simple error in the docs.  Should I just close the PR and post an issue for one of you two to fix instead? 

>$ git clone https://github.com/projecthydra-labs/hyrax.git
Cloning into 'hyrax'...
remote: Counting objects: 75567, done.
remote: Compressing objects: 100% (75/75), done.
remote: Total 75567 (delta 25), reused 0 (delta 0), pack-reused 75489
Receiving objects: 100% (75567/75567), 58.02 MiB | 5.61 MiB/s, done.
Resolving deltas: 100% (51097/51097), done.
MARKs-MB:_no_backup mark$ cd hyrax/
MARKs-MB:hyrax mark$ git checkout fix-readme-typo
Branch fix-readme-typo set up to track remote branch fix-readme-typo from origin.
Switched to a new branch 'fix-readme-typo'
MARKs-MB:hyrax mark$ git rebase master
First, rewinding head to replay your work on top of it...
Applying: Fix application template link
MARKs-MB:hyrax mark$ git status
On branch fix-readme-typo
Your branch and 'origin/fix-readme-typo' have diverged,
and have 7 and 1 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)
nothing to commit, working tree clean
MARKs-MB:hyrax mark$ git push --force
Counting objects: 3, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (2/2), done.
Writing objects: 100% (3/3), 402 bytes | 0 bytes/s, done.
Total 3 (delta 2), reused 1 (delta 1)
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
To https://github.com/projecthydra-labs/hyrax.git
> '+ 3ccd5e525...1f9d7ea33 fix-readme-typo -> fix-readme-typo (forced update)
MARKs-MB:hyrax mark$

mjgiarlo commented:
@mark-dce That's bizarre. I haven't seen that test fail recently, and that code hasn't changed since May.

I'm going to merge your PR anyway. Thanks for submitting it. :)
